library/x509_crt.c

```diff
@@ -2524,6 +2529,142 @@ static int x509_crt_verify_chain(
     }
 }
 
+#ifdef _WIN32
+#ifdef _MSC_VER
+#pragma comment(lib, "ws2_32.lib")
```

Comment:
Shouldn't this be handled by properly setting up project linking dependencies?

Reply:
Please make the changes as you see fit, or ask me to remove _WIN32 support from this patch.

The original patch was a handful of lines of code.  I wrote an entire implementation of inet_pton and tests at the requst of a developer who hasn't found time to review this again.   I'm done expanding this patch.

The CI tests that I can see all pass except for the broken Ubuntu 16.04 containers, which are not related to this patch.

Reply:
You can see in Microsoft's example that my code is valid and one "proper" method:
https://learn.microsoft.com/en-us/windows/win32/winsock/complete-server-code

If mbedtls wants to manage it differently, please do so in a separate commit.